src/form/df.jl

```diff
@@ -0,0 +1,163 @@
+# this file contains (balanced) convexified DistFlow formulation, in W space
+export
+    SOCDFPowerModel, SOCDFForm
+""
+abstract type AbstractDFForm <: AbstractPowerFormulation end
+
+""
+abstract type SOCDFForm <: AbstractDFForm end
+
+""
+const SOCDFPowerModel = GenericPowerModel{SOCDFForm}
+
+"default SOC constructor"
+SOCDFPowerModel(data::Dict{String,Any}; kwargs...) = GenericPowerModel(data, SOCDFForm; kwargs...)
+
+""
+function variable_branch_current(pm::GenericPowerModel{T}, n::Int=pm.cnw; kwargs...) where T <: AbstractDFForm
+    variable_branch_series_current_magnitude_sqr(pm, n; kwargs...)
+end
+
+""
+function variable_branch_flow(pm::GenericPowerModel{T}, n::Int=pm.cnw; kwargs...) where T <: AbstractDFForm
```

Comment:
I think this function can be omitted.  The most generic version works for all GenericPowerModels.

https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/core/variable.jl#L223

Reply:
indeed